pda.sh

```diff
@@ -0,0 +1,42 @@
+package: PDA
+version: "%(tag_basename)s"
+tag: 11.0.7
+source: https://github.com/PascalBoeschoten/pda
+requires:
+  - "GCC-Toolchain:(?!osx)"
+build_requires:
+  - kernel-devel
+  - autotools
+
+--- 
+#!/bin/sh
+
+case $ARCHITECTURE in
```

Comment:
Can't you simply put `PDA:slc7.*` where you actually use this? You would then be guaranteed that you are on slc7 and don't need the case.